test/integration/targets/wait_for/tasks/main.yml

```diff
@@ -62,13 +62,36 @@
     search_regex: completed
     timeout: 20
   register: waitfor
-- name: verify test wait for port timeout
+
+- name: verify test wait for keyword in file
   assert:
     that:
-      - waitfor is successful
-      - "waitfor.search_regex == 'completed'"
-      - waitfor.elapsed >= 5
-      - waitfor.elapsed <= 15
+    - waitfor is successful
+    - "waitfor.search_regex == 'completed'"
+    - waitfor.elapsed >= 5
+    - waitfor.elapsed <= 15
+
+- name: setup write keyword to file after 10s
+  shell: rm -f /tmp/wait_for_keyword && sleep 10 && echo "completed data 123" > /tmp/wait_for_keyword
```

Comment:
Use `{{ output_dir }}/` as a scratch directory instead of `/tmp/`.

The `rm` isn't needed when using `output_dir` since it is empty at the start of each test run.

If possible, use a lower sleep time. I realize 10 seconds was probably copied from the other tests, but as we add more tests those add up. If the test works reliably with 3 or 5 seconds, use that instead.